src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final Random random = new Random();
```

Comment:
The `random` field should not be static. Static fields are shared among all instances of a class, but here each `ColorSupplier` should have its own `Random` instance. Consider making it a private non-static field.